.rubocop.yml

```diff
@@ -158,6 +159,11 @@ Lint/ModuleDisclosureDatePresent:
     # Only exploits require disclosure dates, but they can be present in auxiliary modules etc.
     - 'modules/exploits/**/*'
 
+Lint/SideEffectsInNotes:
+  Include:
+    # Only exploits require disclosure dates, but they can be present in auxiliary modules etc.
+    - 'modules/exploits/**/*'
```

Comment:
I think this copy/pastas needs tweaked, notes should exist in auxiliary modules I think 👀

Reply:
Sure, I was wondering about this and I think it definitely makes sense for aux modules to have side effects, could easily imagine it leaving a trail in the logs